substrate/fd

```diff
@@ -42,21 +44,25 @@ namespace substrate
 		constexpr fd_t() noexcept = default;
 		constexpr fd_t(const int32_t fd_) noexcept : fd{fd_} { }
 		fd_t(const char *const fileName, const int flags, const mode_t mode = 0) noexcept :
-			fd{internal::fdopen(fileName, flags, mode)} { }
+			fd{internal::fdopen(fileName, flags, mode)}
+		{ }
```

Comment:
Something for the style guide (oops, sorry!) - the `{ }` of the empty body being on the previous line is our preferred style for this as otherwise we find this looks like a floating pair of braces.